### PR TITLE
How to

### PR DESCRIPTION
**Pull request by rzuritamilla**
Ready to be merged. More how to questions can be added later on.

### CODE REVIEW

File: how_to_document/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+3. Do a `git status` to double check your current branch
+4. if not in the *master* branch, go to it by using: `git checkout master`
+2. Do a `git pull` to make sure that you have the latest version of this repository
+5. Create a new (local) branch to start documenting a specific dataset and/or analysis: `git checkout -b NAMEOFBRANCH`   
+6. Synchronize the local and github repositories by: `git push -u origin NAMEOFBRANCH`   
+7. Create a folder -ideally with the name of the branch- and a README.md file in it  
+  * in github  
+     1. Open github in your browser, go to the repository and branch where you want to work  
+     2. Click on *create new file*   
+     3. In the "prompt" type your folder name then */* and then README.md
```
Comment (romulogoncalves) [inline how_to_document/README.md]: In the "prompt" type your folder name then */* and then README.md  

and then filename, for example README.md

File: how_to_document/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+4. if not in the *master* branch, go to it by using: `git checkout master`
+2. Do a `git pull` to make sure that you have the latest version of this repository
+5. Create a new (local) branch to start documenting a specific dataset and/or analysis: `git checkout -b NAMEOFBRANCH`   
+6. Synchronize the local and github repositories by: `git push -u origin NAMEOFBRANCH`   
+7. Create a folder -ideally with the name of the branch- and a README.md file in it  
+  * in github  
+     1. Open github in your browser, go to the repository and branch where you want to work  
+     2. Click on *create new file*   
+     3. In the "prompt" type your folder name then */* and then README.md  
+     4. push the *commmit* buttom at the bottom of the page (eventually you can also add a commit message here)
```
Comment (romulogoncalves) [inline how_to_document/README.md]: Eventually no really, you should always add a commit message :)

**Comment by rzuritamilla** [comment]
all fixed by https://github.com/phenology/infrastructure/commit/20997c55f453a96e9fce11af8180c97570147afc

**Comment by romulogoncalves** [comment]
Ready for merge

**Pull request merged**